llumnix/backends/utils.py

```diff
@@ -55,8 +56,9 @@ def initialize_cluster(
             "Ray is not installed. Please install Ray to use distributed "
             "serving.")
     # Connect to a ray cluster.
-    ray.init(address=ray_address, ignore_reinit_error=True)
+    ray.init(address=ray_address, ignore_reinit_error=True, namespace='llumnix')
 
```

Comment:
get_placementgroup maybe better for this method.
mv ray.init to ap_server after launch_ray_cluster

Reply:
Get it.